portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Arushi Gupta</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Arushi Gupta</h1>
+      <p>Hey! </p>
+      <img src="images/dp.JPG" style="width:400px;height:400px;"/>
```

Comment:
Probably I am missing something here, but upon running it is giving me this - 
![Screenshot 2020-07-14 at 6 47 12 PM](https://user-images.githubusercontent.com/14831291/87430248-92011480-c602-11ea-81eb-a468b277e521.png)

What am I missing here?